src/rpc/mining.cpp

```diff
@@ -607,7 +607,6 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
     }
 
     UniValue aux(UniValue::VOBJ);
-    aux.pushKV("flags", HexStr(COINBASE_FLAGS.begin(), COINBASE_FLAGS.end()));
```

Comment:
I'm not sure this counts as simple refactoring. Maybe rename the PR and make it a separate commit?

Reply:
Done (renamed title)